cmd/oras/internal/option/applier.go

```diff
@@ -0,0 +1,22 @@
+package option
```

Comment:
Will `options` sound better than `option`?

Reply:
I remember we perfer not to use plurity unless the key word is already taken. Please thumb up(for optionss) or down(for options)to let me know your opinion.

Reply:
It’s flag package so maybe option is ok 

https://pkg.go.dev/flag